styles/helpers/inputs.css

```diff
@@ -140,4 +140,56 @@ title: Form
       @mixin input var(--color-input-background), var(--color-body), var(--color-input-disabled), var(--color-input-error), var(--color-input-error), var(--color-input-error);
     }
   }
+
+  /*---
+  section: Forms
+  title: Input group
+
+  ```example:html
+  <form class="form">
+    <div class="input-group">
+      <input type="text" placeholder="Enter your email">
+      <button class="input-group-btn button">Sign Up</button>
```

Comment:
Then should we not call it button?

On Fri, Sep 9, 2016, 5:46 PM Newton <notifications@github.com> wrote:

> In styles/helpers/inputs.css
> <https://github.com/firstandthird/clientkit/pull/103#discussion_r78267077>
> :
>
> > @@ -140,4 +140,56 @@ title: Form
> >        @mixin input var(--color-input-background), var(--color-body), var(--color-input-disabled), var(--color-input-error), var(--color-input-error), var(--color-input-error);
> >      }
> >    }
> > +
> > +  /*---
> > +  section: Forms
> > +  title: Input group
> > +
> > +  ```example:html
> > +  <form class="form">
> > +    <div class="input-group">
> > +      <input type="text" placeholder="Enter your email">
> > +      <button class="input-group-btn button">Sign Up</button>
>
> spans for addons like currency or things you'd wnat to suffix with. See
> Bootstrap
> <http://v4-alpha.getbootstrap.com/components/input-group/#basic-example>
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/firstandthird/clientkit/pull/103/files/196e6cfabb2131581368d385302fb569b9fbf574#r78267077>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABtIC3HZSFIZ82Bll51BS-eoXyYalf5ks5qof3IgaJpZM4J0QKD>
> .
>